sources/Platform/Linux/LinuxDisplay.cpp

```diff
@@ -57,8 +58,7 @@ static void* g_retainedLibGL = nullptr;
 LinuxSharedX11Display::LinuxSharedX11Display() :
     native_ { XOpenDisplay(nullptr) }
 {
-    if (!native_)
-        throw std::runtime_error("failed to open connection to X server");
+    LLGL_THROW_IF(!native_, std::runtime_error("failed to open connection to X server"));
```

Comment:
For any remaining exceptions inside source files under `sources/` I'd prefer to use `LLGL_TRAP` (see _sources/Core/Exception.h_) or `LLGL_ASSERT` (see _sources/Core/Assertion.h_). It throws an `std::runtime_error` when exceptions are enabled, breaks the debugger, or calls `abort()` and provides a formatted callstack.